data/mil2525d/utilities/military-features-utilities/template-gdb-toolbox/scripts/CreateTemplateGDB/UpdateDomains.py

```diff
@@ -0,0 +1,59 @@
+# ----------------------------------------------------------------------------------
```

Comment:
It is different from what we normally do for GP tools, but since this is not going into the GP repo, and I'm not in charge of it I guess it really doesn't matter as much. If it will go into the GP repo, then we'll have to modify to fit our GP tool pattern.